### DISCUSSION TITLE
5.0 GA Release Date?

### DISCUSSION THREAD

**Discussion by pinkfloydx33** [dotnet/SqlClient#1667] [General]
Is there a new estimate to when 5.0.0 will be GA? The roadmap estimates it would be May, which has now come and gone.

**Comment by ErikEJ** [comment]
Yes, July 15.

**Reply by pinkfloydx33** [reply]
Thanks!

**Reply by pinkfloydx33** [reply]
Any updated estimate?

**Comment by JRahnama** [comment]
@pinkfloydx33 if everything goes on as we have planned the release will be out this week or the most on Monday.

**Comment by JRahnama** [comment]
@pinkfloydx33 GA v5.0 was released on Friday Aug 5th.

**Reply by pinkfloydx33** [reply]
Thanks!
